client/src/App.js

```diff
@@ -1,22 +1,23 @@
-import Header from './components/Header';
-import Home from './pages/Home';
-import Navigation from './components/Navigation';
 import { Route, Switch } from 'react-router-dom';
 import { useState } from 'react';
+import Home from './pages/Home';
+import CreateMentor from './pages/CreateMentor';
+import Header from './components/Header';
+import Navigation from './components/Navigation';
 
 
 function App() {
 
   const [open, setOpen] = useState(false);
 
   return (
-    <div className="App">
+    <div>
       <Navigation open={open} setOpen={setOpen}></Navigation>
```

Comment:
```suggestion
      <Navigation open={open} setOpen={setOpen} />
```

You can use a self closing tag if you have no children for a component